Integrations/CheckPhish/CheckPhish.py

```diff
@@ -0,0 +1,226 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+BASE_URL = demisto.params()['url']
+API_KEY = demisto.params().get('token')
+GOOD_DISP = demisto.params().get('good_disp')
+SUSP_DISP = demisto.params().get('susp_disp')
+BAD_DISP = demisto.params().get('bad_disp')
+USE_SSL = not demisto.params().get('insecure', False)
+
+HEADERS = {
+    'Content-Type': 'application/json',
+    'Accept': 'application/json'
+}
+
+STATUS_SUFFIX = '/status'
+CLEAN_STATUS = 'clean'
+PENDING_STATUS = 'PENDING'
+DONE_STATUS = 'DONE'
+
+DEFAULT_GOOD_DISP = {
+    'clean'
+}
+
+DEFAULT_SUSP_DISP = {
+    'suspicious',
+    'drug_spam'
+}
+
+DEFAULT_BAD_DISP = {
+    'likely_phish',
+    'phish'
+}
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, url, params=None, data=None):
+    res = requests.request(
+        method,
+        url,
+        verify=USE_SSL,
+        params=params,
+        data=data,
+        headers=HEADERS
+    )
+    if res.status_code not in {200}:
+        return_error('Error in API call to CheckPhish [%d] - %s' % (res.status_code, res.reason))
+
+    try:
+        return res.json()
+
+    except ValueError:
+        return_error('Failed to parse response from service,  please try again later')
```

Comment:
please improve the error message, the user does not have anything to do with that